source/slang/hlsl.meta.slang

```diff
@@ -1395,35 +1395,51 @@ __generic<T : __BuiltinType> T QuadReadAcrossDiagonal(T localValue);
 __generic<T : __BuiltinType, let N : int> vector<T,N> QuadReadAcrossDiagonal(vector<T,N> localValue);
 __generic<T : __BuiltinType, let N : int, let M : int> matrix<T,N,M> QuadReadAcrossDiagonal(matrix<T,N,M> localValue);
 
-__generic<T : __BuiltinIntegerType> T WaveActiveBitAnd(T expr);
+__generic<T : __BuiltinIntegerType>
+__target_intrinsic(cuda, "_waveAnd(__activemask(), $0)")
```

Comment:
Is there a specific reason to favor this lowering vs. just having the CUDA prelude define a `WaveActiveBitAnd` function that uses `__activemask()` in its definition? Is the `_waveAnd` function going to be used elsewhere in the prelude?

Reply:
It was as I believed I could do the Prefix versions by anding with __lanemask_lt(). That's wrong in two ways though. 

1) __lanemask_lt() isn't an intrinsic it seems(!), even though we have this example https://devblogs.nvidia.com/cuda-pro-tip-optimized-filtering-warp-aggregated-atomics/
2) All _sync functions must be invoked by all threads in the mask, with the *same* mask otherwise undefined results. So that wouldn't work anyway.

 So may make sense to remove now.